reservation_units/models.py

```diff
@@ -634,6 +634,100 @@ def state(self) -> ReservationUnitState:
         return ReservationUnitStateHelper.get_state(self)
 
 
+class ReservationUnitPricing(models.Model):
+
+    begins = models.DateField(
+        verbose_name=_("Date when price is activated"),
+        null=False,
+        blank=False,
+        help_text="When pricing is activated",
+    )
+
+    pricing_type = models.CharField(
+        max_length=20,
+        verbose_name=_("Pricing type"),
+        choices=PricingType.choices,
+        blank=True,
+        null=True,
+        help_text="What kind of pricing types are available with this reservation unit.",
+    )
+
+    PRICE_UNIT_PER_15_MINS = "per_15_mins"
```

Comment:
we could move all of these as separate TextChoices or enums

Reply:
Done! I did this refactoring in a separate PR. Otherwise things started to look too confusing for my taste.